app/src/MarketTable/CategoricalMarketTable/index.js

```diff
@@ -144,6 +145,7 @@ const MarketTable = ({
             key={market.conditionId}
             lmsrState={lmsrState}
             headings={headings}
+            tradeHistory={tradeHistory}
```

Comment:
Cannot trace tradeHistory after this line. React developer tools show it to me until "MarketTableComponent" and below that level is always passed as `undefined`

CategoricalMarket Graphs are not draw.